drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/report.rb

```diff
@@ -133,11 +136,7 @@ def enrollment_scope
         preload(:enrollment, :client).
         entry.
         open_between(start_date: filter.start_date, end_date: filter.end_date)
-      filter.apply(scope, report_scope_source)
-    end
-
-    def report_scope_source
-      GrdaWarehouse::ServiceHistoryEnrollment.entry
+      scope.in_project(GrdaWarehouse::Hud::Project.where(id: filter.effective_project_ids))
```

Comment:
removing `filter.apply` means you're only obeying date and project_ids if I'm reading this correctly.  Are there other options on the form?

Reply:
Looks like funding sources at the very least.